Sources/Starknet/Data/Transaction/Data/StarknetTransactionVersion.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public enum StarknetTransactionVersion: String, Codable {
+    case v0 = "0x0"
+    case v1 = "0x1"
+    case v1Query = "0x100000000000000000000000000000001"
+    case v2 = "0x2"
+    case v2Query = "0x100000000000000000000000000000002"
```

Comment:
Should we remove `v2Query` since we don't support sending declares atm?

Reply:
no, let's leave this